### PR TITLE
Fix 415 error

### PR DESCRIPTION
**Pull request by ronenfe**

### CODE REVIEW

File: Aws4Signer/AWS4RequestSigner.cs [hunk 1]
```diff
@@ -7,12 +7,12 @@
 using System.Threading.Tasks;
 using System.Web;
 
-namespace Aws4RequestSigner
+namespace AWS4RequestSigner
```
Comment (tsibelman) [inline Aws4Signer/AWS4RequestSigner.cs]: Please preserve name space it would be a breaking change.

File: Aws4Signer/Aws4RequestSigner.csproj [hunk 2]
```diff
@@ -16,7 +15,7 @@
     <RepositoryType>git</RepositoryType>
     <PackageTags>aws</PackageTags>
     <LangVersion>7.3</LangVersion>
-    <SignAssembly>true</SignAssembly>
+    <SignAssembly>false</SignAssembly>
```
Comment (tsibelman) [inline Aws4Signer/Aws4RequestSigner.csproj]: Revert to true

**Review by tsibelman — CHANGES_REQUESTED** [review]

**Comment by tsibelman** [comment]
Increase assembly version

**Comment by ronenfe** [comment]
done

**Review by tsibelman — APPROVED** [review]

**Review by tsibelman — APPROVED** [review]

**Pull request merged**